.github/workflows/cmake.yml

```diff
@@ -170,6 +170,7 @@ jobs:
           {name: HIP, triplet: amdgcn-amd-amdhsa, platform: ""},
           {name: L0, triplet: spir64, platform: ""},
           {name: OPENCL, triplet: spir64, platform: "Intel(R) OpenCL"}
+          {name: NATIVECPU, triplet: spir64, platform: ""}
```

Comment:
just want to confirm if you're aware - the `name` is also used as a label for self-hosted runner (see `runs-on: ${{matrix.adapter.name}}` below). I don't believe we have such a runner available at the moment.

Reply:
ok, the new runner should be up and running now - please verify if it works as needed 😉

Reply:
Thank you! I started a new job but realised the label for the runner is `NATIVECPU`, the naming in the workflow and CMake is `NATIVE_CPU`. Could you please edit the runner label? Thanks!

Reply:
Done